dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/CostStatisticMetric.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.core.metrics;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cost_metric")
+@Schema(name = "CostMetric", description = "This describes aggregated cost metrics for workflow executions in USD.")
+public class CostStatisticMetric extends StatisticMetric {
+    public static final String UNIT = "USD";
+
+    public CostStatisticMetric() {
+    }
+
+    @JsonCreator
+    public CostStatisticMetric(
+            @JsonProperty("minimum") Double minimum,
```

Comment:
If I understand correctly, the library uses `Money` which is essentially a `BigDecimal`
Does it make sense to pre-emptively  store minimum, maximum, and average as one of those two types in case we need to calculate overall statistics?

Reply:
Based on your comment in https://github.com/dockstore/dockstore-support/pull/471#discussion_r1255929547, I think the answer is yes because we do use these values to calculate overall statistics

Reply:
On closer inspection of the Java Money examples, their examples do use doubles when creating the Money objects [here](https://github.com/JavaMoney/jsr354-ri/blob/master/moneta-core/src/main/asciidoc/userguide.adoc#22-monetary-amounts), so I think it's simplest to keep these as a double